src/nodes.coffee

```diff
@@ -1431,9 +1430,8 @@ exports.Param = class Param extends Base
     return @reference if @reference
     node = @name
     if node.this
-      node = node.properties[0].name
-      if node.value.reserved
-        node = new Literal o.scope.freeVariable node.value
+      name = "at_#{node.properties[0].name.value}"
```

Comment:
First I didn’t. But then I ran into this issue: `(@_slice)->`. What should we do then?